src/helpers/get_metrics_from_df.py

```diff
@@ -1,26 +1,34 @@
 from sklearn.metrics import mean_absolute_error, mean_squared_error, mean_absolute_percentage_error
 import pandas as pd
 
-
-def get_metrics_from_df(df, target_column: str, required_metrics: list = ["mae", "mse"]):
+def get_metrics_from_df(df, observation_column: str, required_metrics: list = ["mae", "mse"]):
     """
-    Calculate specified metrics for each forecast column in a DataFrame compared to a target column.
+    Calculate specified metrics for each forecast column in a DataFrame compared to an observation column.
```

Comment:
What's the reason behind this name change?

Reply:
Because the original name is ["Observation"]